src/include/sof/audio/kpb.h

```diff
@@ -61,6 +61,7 @@ enum kpb_state {
 	KPB_STATE_PREPARING,
 	KPB_STATE_RUN,
 	KPB_STATE_BUFFERING,
+	KPB_STATE_INIT_DRAINING,
```

Comment:
I will really want a better description of *WHY* this is needed. Also, on the long term adding a state diagram with some nice explanations in the documentation project for the KPB component would be really nice.